src/pip/_internal/req/req_set.py

```diff
@@ -16,9 +16,6 @@ def __init__(self,
                  require_hashes=False, target_dir=None, use_user_site=False,
                  pycompile=True):
         """Create a RequirementSet.
-
```

Comment:
This is an unrelated (albeit obvious) change. It'd be nice if you revert this hunk and make a new PR for it.

Reply:
This was 396abfe7fbcd1552dfca753e3b5d2b9f85bad89d done by @alex. I would need to get his consent before removing it.

Reply:
Uhhh, no clue, feel free to ignore whatever I did.

Reply:
I will remove this then.